documentation/faq/70-packages.md

```diff
@@ -2,13 +2,14 @@
 question: How do I fix the error I'm getting trying to include a package?
 ---
 
-Most of these issues come from Vite trying to deal with non-ESM libraries. You may find it helpful to search for your error message in [the Vite issue tracker](https://github.com/vitejs/vite/issues). 
+Most of these issues come from Vite trying to deal with non-ESM libraries. You may find it helpful to search for your error message in [the Vite issue tracker](https://github.com/vitejs/vite/issues).
 
 There are a number of known Vite issues, which cause errors in the following circumstances:
+
 - [packages which use `exports` instead of `module.exports`](https://github.com/vitejs/vite/issues/2579).
```

Comment:
The issue title suggests it's just `exports` vs `module.exports` but they're actually both broken if you read down later on the thread and I've confirmed that

```suggestion
- [CommonJS packages](https://github.com/vitejs/vite/issues/2579).
```